test/integration/test_events.py

```diff
@@ -54,6 +57,24 @@ def test_basic_serializeable():
     json.dumps(events)
 
 
+def test_event_omission():
+    tdir = tempfile.mkdtemp()
+    r = run(private_data_dir=tdir,
+            inventory="localhost ansible_connection=local",
+            omit_event_data=True,
+            playbook=[{'hosts': 'all', 'gather_facts': False, 'tasks': [{'debug': {'msg': "test"}}]}])
+    assert not any([x['event_data'] for x in r.events])
```

Comment:
Do we have tests that verify the `event_data` key for the inverse (when `omit_event_data` is False)?

Reply:
https://github.com/ansible/ansible-runner/blob/master/test/integration/test_events.py#L41

Since we tend *not* to rely on this data internally we don't spend much time validating its internals.